<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>361</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="boardStatusBar">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>悔棋</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索深度</string>
    </property>
   </widget>
   <widget class="QComboBox" name="searchDeepCombBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="diceButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>PC掷骰子</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startMatchButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>590</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>开始比赛</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>我方颜色</string>
    </property>
   </widget>
   <widget class="QComboBox" name="setOurColorComboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>先手</string>
    </property>
   </widget>
   <widget class="QComboBox" name="setSenteComboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="replayMatchButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>590</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>重新开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="useGivenDiceButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>590</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>指定数行走</string>
    </property>
   </widget>
   <widget class="QComboBox" name="setDiceComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>指定骰子数目</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
